.github/workflows/test-workflow.yml

```diff
@@ -77,10 +77,10 @@ jobs:
         uses: step-security/publish-unit-test-result-action@4519d7c9f71dd765f8bbb98626268780f23bab28 # v2.17.0
         if: ${{ !cancelled() }}
         with:
-          check_run_disabled: true
+          check_name: '' # Set to empty to disable check run
```

Comment:
We should enable the checks. They were previously disabled due to a bug in ARC, but that has now been resolved.

Reply:
I've enabled the checks and it passed.